base/src/main/java/io/spine/validate/RangeConstraint.java

```diff
@@ -111,23 +91,15 @@ private EdgeValues(StringDescribedNumber leftEdge, StringDescribedNumber rightEd
          *         right edge of the range
          * @return a new instance of {@code EdgeValues}
          */
-        private static EdgeValues of(String leftEdge, String rightEdge) throws IllegalStateException {
-            StringDescribedNumber left = new StringDescribedNumber(leftEdge);
-            StringDescribedNumber right = new StringDescribedNumber(rightEdge);
+        private static EdgeValues of(String leftEdge, String rightEdge) throws
```

Comment:
Please have `throws` on new lines when it does not fit along with an exception name.